### ISSUE TITLE
Sector codelist is not up to date

### ISSUE THREAD

**Issue by stevieflow** [devinit/D-Portal#615]
eg: https://d-portal.org/ctrack.html#view=act&aid=SE-0-SE-6-5217002501-BGD-12264

This activity has the sector 12264, which is a new'ish sector code for covid-19: 

https://iatistandard.org/en/iati-standard/203/codelists/sector/

![DAC-5-Digit-Sector-iatistandard-org](https://user-images.githubusercontent.com/1209944/112667321-0b6d2a00-8e55-11eb-8f13-81063efeb906.png)

**Comment by notshi** [comment]
Thanks for raising, @stevieflow - this is now live.

Since the last major update which included the new SAVi, there are still a few remaining things that needs looking after but with limited development time allocated, we were not able to. This meant that we were still pulling in codelists manually as we wanted to make sure code changes from IATI weren't affecting the new system in d-portal.

We have updated the script and codelists are now updated nightly. However, do let us know if you find anything amiss.
